SECURITY.md

```diff
@@ -0,0 +1,39 @@
+# Security Policy
+
+## Supported Versions
+
+Redis Stack targets specific versions of Redis releases.
+
+We generally backport security issues to a single previous major version,
+unless this is not possible or feasible with a reasonable effort.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 7.0.x   | :white_check_mark: |
+| 6.2.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you believe you’ve discovered a serious vulnerability, please contact the
+Redis core team at redis@redis.io. We will evaluate your report and if
+necessary issue a fix and an advisory. If the issue was previously undisclosed,
+we’ll also mention your name in the credits.
+
+## Responsible Disclosure
```

Comment:
Shoudl this apply to Redis Stack? Not sure.

Reply:
I generally think security researchers want this. I see this ask all time across other projects, and repositories. It seems consistent with common practice *and* we have the release drafter for this.